src/DataGrid.tsx

```diff
@@ -119,7 +119,7 @@ export interface DataGridProps<R, K extends keyof R, SR = unknown> {
   /** Function called whenever a row is clicked */
   onRowClick?: (rowIdx: number, row: R, column: CalculatedColumn<R, SR>) => void;
   /** Called when the grid is scrolled */
-  onScroll?: (scrollPosition: ScrollPosition) => void;
+  onScroll?: (scrollPosition: ScrollPosition, event: React.UIEvent<HTMLDivElement>) => void;
```

Comment:
We can read `scrollPosition` from the event object so `onScroll(event)` can be used. Wdyt?

Reply:
I agree with @amanmahajan7, let's just pass the event only.